source/databricks/calculation_engine/tests/calculation/energy/grid_loss_responsible_factories.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+from pyspark.sql import Row, SparkSession
+
+from package.calculation.preparation.grid_loss_responsible import (
+    grid_loss_responsible_schema,
+    GridLossResponsible,
+)
+from package.codelists import MeteringPointType
+from package.constants import Colname
+
+DEFAULT_METERING_POINT_ID = "1234567890123"
+DEFAULT_GRID_AREA = "100"
+DEFAULT_FROM_DATE = datetime.strptime("2020-01-01T00:00:00+0000", "%Y-%m-%dT%H:%M:%S%z")
+DEFAULT_TO_DATE = datetime.strptime("2020-01-01T01:00:00+0000", "%Y-%m-%dT%H:%M:%S%z")
+DEFAULT_METERING_POINT_TYPE = MeteringPointType.CONSUMPTION
+DEFAULT_ENERGY_SUPPLIER = "S1"
```

Comment:
How about a (possibly) valid value?
```suggestion
DEFAULT_ENERGY_SUPPLIER = "1234567890123"
```